scripts/remote/controller/perf-analysis.js

```diff
@@ -0,0 +1,93 @@
+/*
+Computes performance data summaries.
+
+Performance test results are spead across many flat files, each with
+timing data. This module analyzes those files to build tabular data,
+and renders that data.
+*/
+
+const fs = require('fs');
+const path = require('path');
+
+const { table } = require('table');
+
+const { lets } = require('./lib.js');
+
+
+// Analysis: Convert result text files to relevant metadata and content.
+/////////////////////////////////////////////////////////////////////////////////
+
+const analyze = (dir) =>
+      Array
+      .from(fs.readdirSync(dir))
+      .filter((entry) => !fs.lstatSync(path.resolve(dir, entry)).isDirectory())
+      .map((entry) => {
+          const absolutePath = path.resolve(dir, entry);
+
+          return {
+              path: absolutePath,
+              name: extractTestName(absolutePath),
+              width: extractWidth(absolutePath),
+              exec_time: extractExecTime(absolutePath),
+              variant: extractVariant(absolutePath),
+          };
+      });
+
+// Pash --width is in the file name
+const extractWidth = (p) =>
+      parseInt((path.basename(p, path.extname(p)).match(/_(\d+)_/) || [])[1]);
+
+const extractVariant = (p) =>
+      (p.match(/_distr_?([^\.]+)/) || [])[1];
```

Comment:
I can't issue a change request on my own PR, so I'll just flag this here.

This pattern looks for test variants that strictly follow `_distr_?`, which means `_seq` is not seen as a test variant. It might be better to parse from right to left as @angelhof mentioned in #108.